python/mujoco_mpc/demos/filter.py

```diff
@@ -12,87 +12,107 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 import matplotlib.pyplot as plt
 import mediapy as media
 import mujoco
-import numpy as np
-
 from mujoco_mpc import filter as filter_lib
+import numpy as np
 
 # %matplotlib inline
-
 # %%
 xml = """
-<mujoco model="Particle1D">
+<mujoco>
+  <option timestep="0.005"/>
+  <default>
+    <geom solimp="0 0.95 0.001"/>
+  </default>
+
   <visual>
     <headlight ambient=".4 .4 .4" diffuse=".8 .8 .8" specular="0.1 0.1 0.1"/>
     <map znear=".01"/>
     <quality shadowsize="2048"/>
+    <global offheight="1024" offwidth="1024"/>
   </visual>
 
-  <custom>
-    <!-- filters: ground truth (0), EKF (1), UKF (2), batch (3) -->
-    <numeric name="estimator" data="2" />
-  </custom>
-
-  <default>
-    <geom solimp="0 0.95 0.001"/>
-  </default>
-
   <asset>
     <texture name="skybox" type="skybox" builtin="gradient" rgb1="0 0 0" rgb2="0 0 0"
              width="800" height="800" mark="random" markrgb="0 0 0"/>
   </asset>
 
+  <custom>
+    <numeric name="estimator" data="1" />
+    <numeric name="estimator_configuration_length" data="12" />
```

Comment:
we can probably remove this line since the default filter is EKF and wont use this field.